server.js

```diff
@@ -4,6 +4,19 @@ const JSONStream = require("JSONStream");
 const redis = require("redis");
 const EventEmitter = require("events");
 const { promisify } = require("util");
+const args = require('yargs').argv;
+
+const { live, file } = args;
+if( live === undefined && file === undefined){
+  console.log("No flag provided. Please provide a valid flag as instucted below.")
+  console.log("Usage: yarn start --live <interface-name> | --file <path-to-pcap-filename>");
+  process.exit(1)
+}
+if( live && file ){
```

Comment:
This is correct - but this is treating both `--live` and `--file` as flags, but they should take extra parameter (mandatory) and used that.

Reply:
Okay, I see.
Will go through the documentation and try to make required changes.